docs/api_static/xrtpy.response.channel.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+`xrtpy.response.channel`
+========================
```

Comment:
The files in `docs/api_static` are to make sure that the functions & classes in these modules show up in the online docs.  It's described more in [PlasmaPy's documentation guide](https://docs.plasmapy.org/en/latest/development/doc_guide.html).